src/Extensions/UserDefinedFormControllerExtension.php

```diff
@@ -112,14 +125,20 @@ public function getFilePath(File $record, string $tempPath = ''): ?string
 
         $dbFile = $record->File;
 
-        $protectedPath = sprintf(
-            '%s/.protected/%s',
-            $basePath,
-            $flysystem->getMetadata(
```

Comment:
`getMetadata` no longer returns the `path` in SS5 so we have to use `getAsURL` instead